types/errors/errors.go

```diff
@@ -78,11 +77,11 @@ var (
 	// ErrWrongPassword defines an error when the key password is invalid.
 	ErrWrongPassword = errorsmod.Register(RootCodespace, 23, "invalid account password")
 
-	// ErrorInvalidSigner defines an error when the tx intended signer does not match the given signer.
-	ErrorInvalidSigner = errorsmod.Register(RootCodespace, 24, "tx intended signer does not match the given signer")
+	// ErrInvalidSigner defines an error when the tx intended signer does not match the given signer.
+	ErrInvalidSigner = errorsmod.Register(RootCodespace, 24, "tx intended signer does not match the given signer")
 
-	// ErrorInvalidGasAdjustment defines an error for an invalid gas adjustment
-	ErrorInvalidGasAdjustment = errorsmod.Register(RootCodespace, 25, "invalid gas adjustment")
+	// ErrInvalidGasAdjustment defines an error for an invalid gas adjustment
+	ErrInvalidGasAdjustment = errorsmod.Register(RootCodespace, 25, "invalid gas adjustment")
```

Comment:
These names should not change as users rely on these error types - let's keep this PR to modifying docs only